pallets/clients-info/src/default_weights.rs

```diff
@@ -0,0 +1,96 @@
+
+//! Autogenerated weights for clients_info
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2023-09-13, STEPS: `100`, REPEAT: `10`, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! WORST CASE MAP SIZE: `1000000`
+//! HOSTNAME: `pop-os`, CPU: `Intel(R) Core(TM) i7-7700K CPU @ 4.20GHz`
+//! EXECUTION: None, WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 1024
+
+// Executed Command:
+// ./target/release/spacewalk-standalone
+// benchmark
+// pallet
+// --chain=dev
+// --pallet=clients-info
+// --extrinsic=*
+// --steps=100
+// --repeat=10
+// --wasm-execution=compiled
+// --output=pallets/clients-info/src/default_weights.rs
+// --template=./.maintain/frame-weight-template.hbs
+
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+#![allow(missing_docs)]
+
+use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
+use core::marker::PhantomData;
+
+/// Weight functions needed for clients_info.
+pub trait WeightInfo {
+	fn set_current_client_release(n: u32, u: u32, ) -> Weight;
+	fn set_pending_client_release(n: u32, u: u32, ) -> Weight;
+}
+
+/// Weights for clients_info using the Substrate node and recommended hardware.
+pub struct SubstrateWeight<T>(PhantomData<T>);
+impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
+	/// Storage: ClientsInfo CurrentClientReleases (r:0 w:1)
+	/// Proof: ClientsInfo CurrentClientReleases (max_values: None, max_size: Some(562), added: 3037, mode: MaxEncodedLen)
+	/// The range of component `n` is `[0, 255]`.
+	/// The range of component `u` is `[0, 255]`.
+	fn set_current_client_release(_n: u32, _u: u32, ) -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `0`
+		//  Estimated: `0`
```

Comment:
@gianfra-t any idea why these seem to be ignored in our benchmarks? I can see in interlay's weights that the `n` and `u` parameters are actually used for calculating the weights. 🤔

Reply:
I don't know! I will try running the command interlay used to see if it has to do with it.

Reply:
Actually, by re running the command, every new run seems to be random in how it tests (or not) each variable 🤔

Reply:
So you are saying that sometimes the variables are included and sometimes they are not? 😶

Reply:
Okay, I believe that what is going on is that in some scenarios the time influence of the parameter is near 0 so it gets ignored.

Reply:
Ah I see, thanks for looking into it again 👍